public/app/shared/toolHrefDirective.js

```diff
@@ -48,8 +48,7 @@
 
       'link': function(scope, elem, attrs) {
         scope.$watch('id', function() {
-          // 'id' and 'referring-tool' attributes may specify either a variable name (the evaluated value
-          // of which appears under 'scope'), or a string literal (which appears under 'attrs').
+          // Examples of 'id': asset id, user id, view name as listed in app.states
```

Comment:
@johncrossman I originally put in the comment above because it took me a while to figure out the `scope || attrs` pattern where the directive was happy to accept either a variable name or a string literal in the referring HTML attribute. Do you think there's value in keeping it?